docs/Installation/Installation.md

```diff
@@ -123,22 +123,24 @@ To build with the docker image:
 
    You will end up in a bash shell in the docker container,
    as root (you need to be root).
-   Within the container, the files in SPECTRE_ROOT are available and
-   Charm++ is installed in `/work/charm`. For the following steps, stay inside
-   the docker container as root.
+   Within the container, the files in SPECTRE_ROOT are available and Charm++ is
+   installed in `/work/charm_6_10_2`. For the following steps, stay inside the
+   docker container as root.
 4. Make a build directory somewhere inside the container, e.g.
    `/work/spectre-build-gcc`, and `cd` into it.
```

Comment:
The advice for where to put the build directory is inconsistent between docker and singularity -- the docker version suggests putting it in the container's filesystem, which will be lost when the container is discarded, and the singularity version suggests putting it as a subdirectory of the source tree.
Is there a good reason for that difference, or should we make them consistent?

Reply:
Good question! Singularity being Linux-only makes things a lot easier because file synchronization between the host and the container is basically (or maybe actually) instant. With Docker on macOS having the build directory on the host will cause you to have rather bad compile times, but maybe that has improved since I last used it (admittedly like 3-4 years ago). @nilsleiffischer do you have a recommendation here?

Reply:
On macOS with Docker I recommend to put the build directory inside the container since the file system sync is still painfully slow and there's no reason to expose the build dir to the host (it's unusable by the host anyway). Output from runs can be shared with the host through other mounts. There's a note a few lines up on appending the `:delegated` flag when mounting volumes to improve performance.

Reply:
Gotcha, thanks for the explanation -- I've usually built in the mounted host dir, but on linux I think that operation is transparent.
I'm happy with the advice as-is then.